sdk/src/validator.rs

```diff
@@ -59,24 +60,28 @@ impl CoseValidator for DummyValidator {
 // • PS512 (RSASSA-PSS using SHA-512 and MGF1 with SHA-512)
 // • RS256	RSASSA-PKCS1-v1_5 using SHA-256
 // • RS384	RSASSA-PKCS1-v1_5 using SHA-384
-// • RS512	RSASSA-PKCS1-v1_5 using SHA-512
+// • RS512	RSASSA-PKCS1-v1_5 using SHA-512 // validation not supported by Ring
 // • ED25519 Edwards Curve ED25519
```

Comment:
All RSAxxx have been deprecated by C2PA so it OK to remove these.

Reply:
Excellent, I'll remove that.